### DISCUSSION TITLE
Adding a Profile field for: location, gender designation(pronouns), birthday, website

### DISCUSSION THREAD

**Discussion by ghost** [bluesky-social/atproto#2142] [Bluesky Lexicons]
I think this is likely a protocol (lexicon)  change
 -to add the ability to add location, gender designation birthday and a website to the bio in pre allocated spots.

this is something people are accustom to on social media,  and gives profiles some consistency between users

adding a gender field/pronoun designation (sooner rather than later) i can also see minimizing potential conflicts or contention with some users  

if this is not too complex of a change, i see it having a large scale benefit and possibly a quick turn around for 3rd party devs to also implement. 

adding this field could also be a reason to fix long bios (https://github.com/bluesky-social/social-app/issues/2220) 

before i make this too complicated - id say one additional bonus that would make many people happy is having an additional "tip" field where users could link ko-fi, patreon, gofundme,  venmo, cashapps, or other tipping services ppl use.

**Comment by mackuba** [comment]
I wonder if this should be a list of known fields, or just some number of fields where you can put your own name and value like on Mastodon?

**Reply by ghost** [reply]
i think having some structure i think ie. list of known fields, is the best for keeping it cohesive between apps, and profiles.   

 that was my big gripe with mastodon and many link in bio services is that their was no standard layout which sometimes made it harder to find the info i was looking for. 
ideally imo, 
ie.  gender as one spot, location as one spot, links as one spot (tip as a bonus) as another spot,  and then a  one or many for links, --- giving them their own fields i think helps keep it cohesive  (and may make it easier to moderate for slurs potentially)  but not sure,

**Comment by bnewbold** [comment]
This is definitely something we'll want to address, but most of the team is very heads down this month, it will probably a bit before we iterator on the profile record schema.

**Reply by ghost** [reply]
> This is definitely something we'll want to address, but most of the team is very heads down this month, it will probably a bit before we iterator on the profile record schema.

Thank you 🙏

**Comment by aendra-rininsland** [comment]
Bit of an aside:

I would love this; particularly if it allows [RelMeAuth](https://microformats.org/wiki/RelMeAuth) a la Mastodon, it would make verifying users like e.g. journalists (👋 ) vastly easier. While handle-based verification via DNS (such as how I add new accounts to the "📰 News" feed) is really useful for organisations, at present I really have no ability to verify journalists for my "Verified Journalists" list other than reaching out via alternative channels to confirm ownership of any given handle.

Using RelMeAuth would be VASTLY simpler than using the [`.well-known/atproto-did`](https://bsky.social/about/blog/4-28-2023-domain-handle-tutorial#:~:text=How%20can%20I%20set%20and%20manage%20multiple%20subdomain%20handles%3F) approach, which I suspect even orgs who've gone through the effort of using their domain as handle will struggle to do (my employer has more journalists active on Bluesky than any other newspaper AFAICT but speaking from experience, setting up some system that would allow them to have e.g. `.ft.com` handles would be incredibly challenging both in terms of implementation but also governance).

**Reply by Zackaryia** [reply]
RelMeAuth would be nice to implement for bio links but for handles I am hesitant because checking if you own a handle via downloading a webpage that is potentially extremely large, that could block you for bot traffic, and other such issues, then parsing for a specific string could cause issues for something as crucial as user handles. However I would like to see this for bios.

**Reply by aendra-rininsland** [reply]
@Zackaryia Unless I've really misunderstood something, I'm only suggesting this for bio links.
